scripts/bump-version/main.ts

```diff
@@ -0,0 +1,57 @@
+import * as path from "@std/path";
+
+import { REPO_ROOT } from "@/constants.ts";
+import { CalVer } from "@/bump-version/calver.ts";
+import { parseArgs } from "@std/cli";
+
+const args = parseArgs(Deno.args, { boolean: ["all"] });
+
+if (!Deno.env.get("CI") && !args.all) {
+  throw new Error(
+    "This script is not intended to be used manually. Userstyle versions are automatically bumped after pull requests are merged.",
```

Comment:
Bit of a nit but we should prefer being direct in all documentation / try our best to be. Thoughts on the wording below?

```suggestion
    "This script is intended to be used in CI. Userstyle versions are automatically bumped after pull requests are merged.",
```

Reply:
100%, this reads much better. Thanks!